src/utils/models/User.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../../utils/db');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+    allowNull: false,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+}, {
+  tableName: 'users',
+  updatedAt: false,
+  createdAt: false,
```

Comment:
I also mark this to false, but mentors mark that better leave this columns to be created